CHANGELOG.md

```diff
@@ -30,7 +30,7 @@ off).
 multi_phase_center objects (it was using the old `phase_center_frame` attribute).
 - Fix a bug where trying to select lsts or lst_ranges on read didn't work for some file
 types.
-
+- Severe performance hit when calling `polnum2str` and its variants for many baselines.
 ### Deprecated
```

Comment:
please add an empty line before the `Deprecated` heading.

Reply:
Ah, done.